src/operator/tensor/elemwise_binary_op_basic.cu

```diff
@@ -36,21 +36,21 @@ NNVM_REGISTER_OP(_backward_add)
   ElemwiseBinaryOp::BackwardUseNoneWithHalf2<gpu, mshadow_op::identity,
     mshadow_op::identity>);
 
-NNVM_REGISTER_OP(_sub)
+NNVM_REGISTER_OP(elemwise_sub)
 .set_attr<FCompute>("FCompute<gpu>", ElemwiseBinaryOp::ComputeWithHalf2<gpu, mshadow::op::minus>);
 
 NNVM_REGISTER_OP(_backward_sub)
 .set_attr<FCompute>("FCompute<gpu>", ElemwiseBinaryOp::BackwardUseNoneWithHalf2<
   gpu, mshadow_op::identity, mshadow_op::negation>);
 
-NNVM_REGISTER_OP(_mul)
+NNVM_REGISTER_OP(elemwise_mul)
 .set_attr<FCompute>("FCompute<gpu>", ElemwiseBinaryOp::ComputeWithHalf2<gpu, mshadow::op::mul>);
 
 NNVM_REGISTER_OP(_backward_mul)
 .set_attr<FCompute>("FCompute<gpu>",
   ElemwiseBinaryOp::BackwardUseInWithHalf2<gpu, mshadow_op::right, mshadow_op::left>);
 
-NNVM_REGISTER_OP(_div)
+NNVM_REGISTER_OP(elemwise_div)
```

Comment:
Doesn't this break backward compatibility? @cjolivier01 @piiswrong

Reply:
This is fixing a bug introduced by chris. _div is renamed in .cc but not in .cu